File: Swap Two Given Positions/SwapTwoGivenPositions.java
public class SwapTwoGivenPositions {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter the number of elements: ");
        int n = sc.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Enter element " + (i + 1) + ": ");
            a[i] = sc.nextInt();
        }
        System.out.print("Enter first position (0-based index): ");
        int pos1 = sc.nextInt();
        System.out.print("Enter second position (0-based index): ");
        int pos2 = sc.nextInt();
        int temp = a[pos1];
        a[pos1] = a[pos2];
        a[pos2] = temp;
        System.out.println("Array after swapping the given positions:");
        for (int i = 0; i < n; i++) {
            System.out.println(a[i]);
        }
    }
}

File: Swap Two Given Positions/swap_two_given_positions.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    int pos1, pos2;
    printf("Enter first position (0-based index): ");
    scanf("%d", &pos1);
    printf("Enter second position (0-based index): ");
    scanf("%d", &pos2);
    int temp = a[pos1];
    a[pos1] = a[pos2];
    a[pos2] = temp;
    printf("Array after swapping the given positions:\n");
    for (int i = 0; i < n; i++) {
        printf("%d\n", a[i]);
    }
    return 0;
}
